docs/architecture/adr-013-per-chain-default-address.md

```diff
@@ -0,0 +1,114 @@
+# ADR 013: Per-chain default external address
+
+## Changelog
+
+- May 18th, 2022: Initial draft;
+
+## Status
+
+DRAFT Not Implemented
+
+## Abstract
+
+This ADR contains the specification of _per-chain default external addresses_, a new feature that will allow users to specify which external address should be used as the default one for each external chain they have linked to their Desmos profile.
+
+## Context
+
+Currently, Desmos allows users to link external chain accounts to their profile by specifying a chain name and an external address associated to such chain. For each chain, each user can link multiple external addresses to the same profile. Although this allows for greater extensibility and usability, there are some occasions in which it might make it harder to properly use this feature. For example, one application might want to deal with a single external address per chain. Right now this is not easy to do as there are many ways in which that single chain link can be chosen among multiple ones (e.g. the first created, or the last one). 
+
+## Decision
+
+We will allow users to specify a _default address_ for each external chain they have connected to their Desmos profile. Initially, when creating an external chain link, the first link will be set as the default one. Later, each user will be able to set a specific link as the default one to be used when dealing with a specific chain.
+
+### Store
+
+Each default chain link will be stored inside the state using the following key:
+```
+DefaultChainLinkPrefix | UserAddress | ChainName | -> ExternalAddress 
+```
+
+This will allow to easily get a default external address based on the user address and chain name, which is going to be the most used query. It will also allow to easily iterate over all default external addresses for each user. 
+
+### `Msg` Service
+In order to allow each user to set their own custom default external addresses, we will need to implement a new message. 
+
+```protobuf
+syntax = "proto3";
+
+// Msg defines the profiles Msg service.
+service Msg {
+  // SetDefaultAddress allows to set a specific external address as the default one for a given chain
+  rpc SetDefaultAddress(MsgSetDefaultAddress) returns (MsgSetDefaultAddressResponse);
```

Comment:
I would suggest to change the name to `SetDefaultExternalAddress` since like this it can be misunderstood with the actual address users have on our chain.